lib/wrapbox/runner/docker.rb

```diff
@@ -15,7 +16,14 @@ class ExecutionError < StandardError; end
 
       def initialize(options)
         @name = options[:name]
-        @container_definition = options[:container_definition]
+        @container_definitions = options[:container_definition] ? [options[:container_definition]] : options[:container_definitions][0]
```

Comment:
I think `[0]` should be removed from `options[:container_definitions][0]`.

Reply:
[imo] `@container_definitions = [options[:container_definition], *options[:container_definitions]].compact`, or raise an error if both of `container_definition` and `container_definitions` are specified.

Reply:
Already fixed.